### ISSUE TITLE
Datatable 508 issue

### ISSUE THREAD

**Issue by MrMcBossman** [primefaces/primevue#2317]
**I'm submitting a ...
```
[x] bug report 
```

**Current behavior**
Datatable fails the google lighthouse accessibility check.  "[aria-*] attributes do not match their roles"

**Expected behavior**
Pass the accessibility check with 100%

**Minimal reproduction of the problem with instructions**
Navigate to https://primefaces.org/primevue/#/datatable and run the google lighthouse tool from the inspect element menu in desktop mode with only accessibility checked. 
The lighthouse errors say "[aria-*] attributes do not match their roles" on the column headers

**What is the motivation / use case for changing the behavior?**
Create a 508 compliant Primevue datatable

**Please tell us about your environment:**
Chrome browser
Vue 3.12.1
Centos 7

* **Vue version:** 3.X
Vue 3

* **PrimeVue version:** 3.4.X
Vue 3.12.1

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
Chrome

**Comment by tugcekucukoglu** [comment]
Duplicate [#1973 ](https://github.com/primefaces/primevue/issues/1973)

All accessibility issues will be fixed in 2022 Q2. Thanks for your report.
